### PR TITLE
UI bugfixing

### PR DESCRIPTION
**Pull request by kprzerwa**
to review: last 2 commits
closes: #73

### CODE REVIEW

File: src/lib/api/date.js [hunk 1]
```diff
@@ -51,5 +51,5 @@ export const toShortDate = date => {
  */
 export function dateFormatter({ col, row }, defaultValue = null) {
   const dateField = _get(row, col.field);
-  return dateField ? toShortDate(fromISO(dateField)) : defaultValue;
+  return dateField ? fromISO(dateField).toLocaleString() : defaultValue;
```
Comment (ntarocco) [inline src/lib/api/date.js]: 👍

File: src/lib/components/CancelModal/CancelModal.js [hunk 2]
```diff
@@ -53,13 +53,20 @@ export default class CancelModal extends Component {
...
       <Modal
         basic
         size="small"
-        trigger={<Button primary content={buttonText} onClick={this.show} />}
+        trigger={
+          <Button
+            primary
+            content={buttonText}
+            onClick={this.show}
+            loading={isLoading}
```
Comment (ntarocco) [inline src/lib/components/CancelModal/CancelModal.js:67]: Could you please:
```suggestion
            loading={isLoading}
            disabled={isLoading}
```
I think we should add to all buttons with a loading state: my fear (did not test it!) is that you can still click while loading? We should maybe test it...

File: src/lib/modules/Loan/actions.js [hunk 3]
```diff
@@ -60,12 +60,7 @@ export const performLoanAction = (
         type: ACTION_SUCCESS,
         payload: response.data,
       });
-
-      dispatch({
-        type: DETAILS_SUCCESS,
-        payload: response.data,
-      });
-
+      dispatch(fetchLoanDetails(response.data.pid));
```
Comment (ntarocco) [inline src/lib/modules/Loan/actions.js:63]:
```suggestion
      dispatch(fetchLoanDetails(response.data.medata.pid));
```
?

I would really love to remove that extra `pid` at the root level

File: src/lib/pages/backoffice/Loan/LoanDetails/LoanActions/LoanActions.js [hunk 4]
```diff
@@ -32,9 +33,10 @@ export default class LoanActions extends Component {
               cancelText="Cancel Loan"
               buttonText="cancel"
               action={cancelAction}
+              isLoading={isLoading}
```
Comment (ntarocco) [inline src/lib/pages/backoffice/Loan/LoanDetails/LoanActions/LoanActions.js:36]: same here for the buttons disabled

File: src/lib/components/CancelModal/CancelModal.js (hunk 2 above)
Comment (kprzerwa) [reply src/lib/components/CancelModal/CancelModal.js:67]: yes, true, good point

**Pull request merged**